problem7.py

```diff
@@ -0,0 +1,53 @@
+"""
+Created on Sun Mar 18 19:05:55 2018
+@author: patricio
+
+1) is_prime: Receives a number and determines if the number is prime.
+    
+2) find_next_prime_num_after: Returns the next prime number that follows
+    the imputed number
+    
+"""
+import time
+
+def is_prime(number):
+    default = True
+    if number in [0,1]:
+        default = False
+    elif number == 2:
+        default = True
+    else:
+        for i in range(2,number//2+1):
+            if number % i == 0:
+                default = False
+                break
+    return default
+
+def find_next_prime_num_after(num):
+    condition_meet = False
+    if num % 2 == 0:
+        k = num//2 #Since we're only checking numbers of the form 2k+1
+    else:
+        k = num//2 + 1
+    num = 2*k + 1 
+    while(condition_meet == False):
+        if(is_prime(num)):
+            condition_meet = True
+        else:
+            k+=1
+            num = 2*k+1 #This reduces numbers of loops by eliminating all even numbers
+    return num
+
+start = time.time()
+
+prime_num = 2
+prime_pos = 1
+
+while(prime_pos < 10001):
```

Comment:
This takes almost 16 seconds to run and it's brute force looking, I should look for a different algorithm.
It might be possible to adapt problem's 10 algorithm to solve this one faster.

Reply:
Problem was that I was looking too high in the is_prime() function, when I adjusted the limit of the algorithm to just look until squared root, the execution time reduced dramatically.